src/main/java/carRacing/controller/CarRacingController.java

```diff
@@ -0,0 +1,22 @@
+package carRacing.controller;
+
+import carRacing.domain.Car;
+import carRacing.domain.CarFactory;
+import carRacing.domain.View;
+import carRacing.dto.UserInput;
+import carRacing.service.CarRacingService;
+
+import java.util.List;
+
+public class CarRacingController {
+
+    private View view = new View();
+    private CarFactory carFactory = new CarFactory();
+    private CarRacingService carRacingService = new CarRacingService();
+
+    public void run() {
+        UserInput userInput = view.getUserInput();
+        List<Car> cars = carFactory.initCars(userInput);
+        carRacingService.run(userInput, cars);
```

Comment:
컨트롤러가 생겼군요. 그런데 이 컨트롤러와 `CarRacingService` 가 하는일이 거의 비슷한것 같아요. 실제로 `CarRacingService#run` 을 이곳에 inline 해도 이전과 거의 다르지 않겠어요.
```suggestion
        List<Car> cars = new CarFactory().initCars(userInput);
        view.declareRace();
        new Game().runRace(userInput, cars);
        view.printWinners(new Judge().findWinners(cars));
```